jupyter_ydoc/ystdin.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+from pycrdt import Map, Text
+
+
+def add_stdin(cell: Map, prompt: str = "", password: bool = False) -> None:
+    """
+    Adds an stdin Map in the cell outputs.
+
+    Schema:
+
+    .. code-block:: json
+
+        {
+            "state": Map[
+                "pending": bool,
```

Comment:
I think there are no non-pending inputs, they get cleared away once the result was submitted.

Reply:
Though maybe there will be fore example if the kernel dies. Should we then clear the input box or keep it with some kind of disconnected icon?

Reply:
By "pending" I meant that the input is in a "being edited" state or a "submitted" state, in which case it cannot be edited again.

Reply:
> Though maybe there will be fore example if the kernel dies. Should we then clear the input box or keep it with some kind of disconnected icon?

In this case I guess we should just remove the `stdin` output from the shared model, because the kernel is not actually waiting for an input anymore, right?